### PR TITLE
fix(config): update misleading missing API key error message

### PR DESCRIPTION
**Pull request by sanderblue**
Resolves: #1065 

Checklist
- [x] update client to v0.55.2 to capture updated messaging
- [x] update "getting started" docs to reflect the most recent changes for how customers can get their API key set up.

### CODE REVIEW

File: website/docs/guides/getting_started.html.markdown [hunk 1]
```diff
@@ -280,4 +230,5 @@ If you ever need to make changes to your configuration, you can run `terraform a
 You can also run `terraform destroy` to tear down your resources if that's ever needed.
 
 [personal_api_key]: https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#personal-api-key
```
Comment (jpvajda) [inline website/docs/guides/getting_started.html.markdown:232]: We seem to be calling this the use key now..  suggest we reference it correctly. 
```suggestion
[user_api_key]: https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#user-api-key
```

File: website/docs/guides/getting_started.html.markdown [hunk 2]
```diff
@@ -13,58 +13,8 @@ description: |-
...
-               "nextCursor": null
-             }
-           }
-         }
-       }
-     }
-     ```
-  1. The value for data→actor→apiAccess→keySearch→keys→key is the key.
-  
+* Locate your Personal API key by following [New Relic's Personal API key docs][personal_api_key]. <br> <small>Note: To to locate or create your API key via our NerdGraph API, please follow these [instructions][personal_api_key_via_nerdgraph].
```
Comment (jpvajda) [inline website/docs/guides/getting_started.html.markdown:16]:
```suggestion
* Locate your User API key by following [New Relic's User API key docs][user_api_key]. <br> <small>Note: To to locate or create your API key via our NerdGraph API, please follow these [instructions][user_api_key_via_nerdgraph].
```

File: website/docs/guides/getting_started.html.markdown [hunk 3]
```diff
@@ -280,4 +230,5 @@ If you ever need to make changes to your configuration, you can run `terraform a
 You can also run `terraform destroy` to tear down your resources if that's ever needed.
 
 [personal_api_key]: https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys#personal-api-key
+[personal_api_key_via_nerdgraph]: https://docs.newrelic.com/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys
```
Comment (jpvajda) [inline website/docs/guides/getting_started.html.markdown:233]:
```suggestion
[user_api_key_via_nerdgraph]: https://docs.newrelic.com/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys
```

**Review by jpvajda — CHANGES_REQUESTED** [review]

**Review by jthurman42 — APPROVED** [review]
:shipit:

**Review by jpvajda — APPROVED** [review]
LGTM

**Pull request merged**